elements/pl-rich-text-editor/pl-rich-text-editor.js

```diff
@@ -62,24 +62,15 @@ class MathFormula extends Embed {
   }
 
   static updateNode(node, value) {
-    // Wait for the page to be fully loaded. Mathjax will not work unless the page is fully loaded.
-    if (document.readyState === 'complete') {
-      let html = MathJax.tex2chtml(value);
+    MathJax.startup.promise.then(async () => {
+      let html = await MathJax.tex2chtmlPromise(value);
```

Comment:
Just did some tests locally. Adding formulas works fine, but when loading an answer that already includes some formulas, the console log includes an error:

<img width="261" alt="image" src="https://user-images.githubusercontent.com/1004907/193691951-708871db-3a5f-4c98-a2e6-dbd304512c3e.png">

I'm playing around to see if I can find the source, but just letting you know. This seems to only affect the question (editable) components, not the submission (read-only) components.

Reply:
@jonatanschroeder I noticed that too, but the problem already exists on master. As it's not a regression, I don't think that should stand in the way of merging this.

The underlying problem is that `quilljs-markdown` assumes that `Quill#getSelection()` will always return a non-null value, but it's explicitly documented as sometimes returning null.

https://github.com/cloverhearts/quilljs-markdown/blob/84c80fee7d94bc2b027660e258137db7252e8f07/src/app.js#L167

Reply:
This seems to be unrelated to this issue. I'll open a new issue for it. It seems to be related to quilljs-markdown, I'll see if I can either fix it or replace it with another equivalent module (there seem to be other more reliable options).